### PR TITLE
add a future work chapter

### PR DESCRIPTION
**Pull request by CaptainSifff**

### CODE REVIEW

File: competencies.md [hunk 1]
```diff
@@ -1081,6 +1081,28 @@ Qualification badges are another venue, that RSEs to find people with the same t
 Structuring and institutionalizing the education and structures for the add-on courses that are also open to others in academia,
 will be topics of a follow-up paper.
 
+# Future Work
+Having the competencies is a first step to find common ground around which to structure 
+curricula, institutions, and teachers in these structures.
+Therefore a diverse set of publications is already in the making.
+Next we will work on how to institutionalize education.
```
Comment (jpthiele) [inline competencies.md:1088]: Next,

File: competencies.md [hunk 2]
```diff
@@ -1081,6 +1081,28 @@ Qualification badges are another venue, that RSEs to find people with the same t
...
+Here we will detail how we organize our institutions and what qualifications our teachers
+have to possess in order to effectively communicate our values.
+We will put forward ideas on how to build up bachelors and masters program,
```
Comment (jpthiele) [inline competencies.md:1091]: Not sure what was used in the rest of the paper but I think it was Master's?

File: competencies.md [hunk 3]
```diff
@@ -1081,6 +1081,28 @@ Qualification badges are another venue, that RSEs to find people with the same t
...
+of which a glimpse can already be found in this paper's appendix.
+We will show how we intend to provide the necessary continuous education for RSEs after graduation.
+This publication is again intentionally free of regional specifics, to also serve
+as a blueprint that other national RSE societies can build upon.
+Another important block is to provide people with online resources for use in their
+courses. This is the  intention of the so-called "survey-publication".
+This work will not be carried out as a traditional publication, but
+this survey of existing resources will be made available as a
+continuously evolving online resource.
+And finally we will formulate the call to action - building on the previous
```
Comment (jpthiele) [inline competencies.md:1101]: finally,

File: competencies.md [hunk 4]
```diff
@@ -1081,6 +1081,28 @@ Qualification badges are another venue, that RSEs to find people with the same t
...
+as a blueprint that other national RSE societies can build upon.
+Another important block is to provide people with online resources for use in their
+courses. This is the  intention of the so-called "survey-publication".
+This work will not be carried out as a traditional publication, but
+this survey of existing resources will be made available as a
+continuously evolving online resource.
+And finally we will formulate the call to action - building on the previous 
+publication on the necessary institutions,
+to lay down what is required to at best support the continuous need
+for young RSEs to support digital science in Germany.
```
Comment (jpthiele) [inline competencies.md:1104]: maybe a 'specifically' somwhere?

**Review by jpthiele — APPROVED** [review]
Very nice outlook!
Comments are only minor typos.

**Comment by CaptainSifff** [comment]
Thanks!

File: competencies.md [hunk 5]
```diff
@@ -1081,6 +1081,28 @@ Qualification badges are another venue, that RSEs to find people with the same t
...
+Next, we will work on how to institutionalize education.
+Here we will detail how we organize our institutions and what qualifications our teachers
+have to possess in order to effectively communicate our values.
+We will put forward ideas on how to build up bachelors and master's program,
+of which a glimpse can already be found in this paper's appendix.
+We will show how we intend to provide the necessary continuous education for RSEs after graduation.
+This publication is again intentionally free of regional specifics, to also serve
+as a blueprint that other national RSE societies can build upon.
+Another important block is to provide people with online resources for use in their
+courses. This is the  intention of the so-called "survey-publication".
```
Comment (mhagdorn) [inline competencies.md:1097]: do we have link yet for this? I guess we will when we are ready to publish

**Comment by mhagdorn** [comment]
I've added some very minor changes - this ties in the other papers quite nicely

**Pull request merged**